libs/core/langchain_core/runnables/graph_mermaid.py

```diff
@@ -291,9 +292,14 @@ async def _render_mermaid_using_pyppeteer(
     img_bytes = await page.screenshot({"fullPage": False})
     await browser.close()
 
+    def write_to_file(path: str, bytes: bytes) -> None:
```

Comment:
is this from a ruff rule?

Reply:
Yes because we are in an async context so we can’t use blocking `open()` and `write()` directly (ruff rule ASYNC)